### PR TITLE
Bug 1135664: configure new masters and servers for swap on ephemeral0

### PR DESCRIPTION
**Pull request by djmitche**

### CODE REVIEW

File: configs/buildbot-master.cloud-init [hunk 1]
```diff
@@ -10,4 +10,7 @@ ssh_pwauth: true
 moz_instance_type: {moz_instance_type}
 # buildbot masters use their first ephemeral disk for swap
 mounts:
- - [ ephemeral0, /mnt, auto, "defaults,noexec" ]
+ - [ ephemeral0, none, swap, sw, 0, 0 ]
+bootcmd:
+ - mkswap /dev/xvdb
```
Comment (rail) [inline configs/buildbot-master.cloud-init]: maybe add a guard to make sure that /dev/xvdb is an ephemeral disk? something like 
mkswap /dev/`curl http://169.254.169.254/latest/meta-data/block-device-mapping/ephemeral0`

Comment (djmitche) [inline configs/buildbot-master.cloud-init]: I'm testing it out now.

Comment (djmitche) [inline configs/buildbot-master.cloud-init]: Well, that doesn't work.  If cloud-init did any logging, I might know why, but it doesn't.

File: configs/buildbot-master.cloud-init [hunk 2]
```diff
@@ -10,4 +10,7 @@ ssh_pwauth: true
 moz_instance_type: {moz_instance_type}
 # buildbot masters use their first ephemeral disk for swap
 mounts:
- - [ ephemeral0, /mnt, auto, "defaults,noexec" ]
+ - [ ephemeral0, none, swap, sw, 0, 0 ]
+bootcmd:
+ - mkswap $(curl http://169.254.169.254/latest/meta-data/block-device-mapping/ephemeral0)
```
Comment (rail) [inline configs/buildbot-master.cloud-init]: I believe you need to add "/dev/" because that URL returns thee basename only ("xvdb")

**Comment by djmitche** [comment]
Ugh, I keep getting bitten by this -- aws_create_instance doesn't use the `.cloud-init` from the configs directory you pass as an argument, it uses the one from the Python source directory.  So, another round of testing.

**Comment by djmitche** [comment]
Roger, re-testing.

**Comment by djmitche** [comment]
Woo,
```
[root@buildbot-master999.bb.releng.use1.mozilla.com ~]# swapon -s
Filename                                Type            Size    Used    Priority
/dev/xvdb                               partition       4188668 0       -1
```

**Pull request merged**